remoteworkitem/trackerquery_repository.go

```diff
@@ -30,7 +30,7 @@ func NewTrackerQueryRepository(db *gorm.DB) *GormTrackerQueryRepository {
 
 // TrackerQueryRepository encapsulate storage & retrieval of tracker queries
 type TrackerQueryRepository interface {
-	repository.Exister
+	CheckExists(ctx context.Context, id string) error
```

Comment:
why do we need to remove? can't it be UUID and I did not find caller for this function in its controller?

Reply:
It needs to be standalone because the tracker query can only handle strings as IDs. @DhritiShikhar correct?

Reply:
oh! okay so maybe trackequery is being used from outside values which are strings. okay I might have missed usage of it.